### PR TITLE
NF: factorize duplicate switch branch

### PR DESCRIPTION
**Pull request by Arthur-Milchior**

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.java [hunk 1]
```diff
@@ -3003,14 +3001,12 @@ public void decrementCounts(@Nullable Card discardCard) {
             mNewCount--;
             break;
         case Consts.QUEUE_TYPE_LRN:
-            mLrnCount --; // -= discardCard.getLeft() / 1000; in sched v1
+            case Consts.QUEUE_TYPE_DAY_LEARN_RELEARN:
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.java:3004]: Fix indentation

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Quick fix, then good to go

**Comment by mikehardy** [comment]
and a conflict now :-)

**Comment by Arthur-Milchior** [comment]
done

**Review by mikehardy — APPROVED** [review]
yep

**Pull request merged**